app/src/main/java/org/wikipedia/views/PageItemView.java

```diff
@@ -229,16 +208,26 @@ private void init() {
             setForeground(AppCompatResources.getDrawable(getContext(), ResourceUtil.getThemedAttributeId(getContext(), R.attr.selectableItemBackground)));
         }
 
-        FeedbackUtil.setToolbarButtonLongPressToast(primaryActionView);
         FeedbackUtil.setToolbarButtonLongPressToast(secondaryActionView);
     }
 
-    private void updateSelectedState() {
-        imageSelectedView.setVisibility(selected ? VISIBLE : GONE);
-        imageView.setVisibility(selected ? INVISIBLE : VISIBLE);
-        // TODO: animate?
-        setBackgroundColor(getThemedColor(getContext(),
-                selected ? R.attr.multi_select_background_color : R.attr.paper_color));
+    public void setImageContainerEndMargin(int dimension) {
```

Comment:
Not sure if it is necessary, I cannot see the difference if I remove this method.

Reply:
Nice catch... this was used in an earlier patch